.editorconfig-checker.json

```diff
@@ -0,0 +1,10 @@
+{
+    "Format": "github-actions",
+    "Exclude": [
+        "osdep/dirent-win.h"
+    ],
+    "Disable": {
+        "IndentSize": true,
+        "MaxLineLength": true
```

Comment:
We could be more strict about this, but sometimes it is just cleaner to let overhanging line be.

Reply:
would be around 5400 with 80 `max_line_length` and 730 changes with 100 `max_line_length`. don't think we want to go down that road here now? maybe at a later point of time if we want to be really strict?

Reply:
Sure, it always depends on the context, sometimes it is just better to have longer line.